upstart/reddit-job-gold_accounting.conf

```diff
@@ -0,0 +1,12 @@
+description "email gold accounting csv monthly"
```

Comment:
Just a note that we'll also need a puppet pull request to make this run on some machine.